kubernetes/ansible-kube/environments/kube/files/invoker.yml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: invoker
+  namespace: openwhisk
+  labels:
+    name: invoker
+spec:
+  replicas: 1
+  serviceName: "invoker"
+  template:
+    metadata:
+      labels:
+        name: invoker
+    spec:
+      restartPolicy: Always
+
+      volumes:
+      - name: cgroup
+        hostPath:
+          path: "/sys/fs/cgroup"
+      - name: runc
+        hostPath:
+          path: "/run/runc"
+      - name: dockerrootdir
+        hostPath:
+          path: "/var/lib/docker/containers"
+      - name: dockersock
+        hostPath:
+          path: "/var/run/docker.sock"
+      - name: apparmor
+        hostPath:
+          path: "/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
+
+      containers:
+      - name: invoker
+        imagePullPolicy: IfNotPresent
+        image: danlavine/whisk_kube_invoker
```

Comment:
Nice. We took a similar approach for DCOS Packages. See https://github.com/openwhisk/openwhisk-devtools/pull/20/files#diff-673d38070852220704426f492fdb101cR95 

Docker command is
```
/invoker/bin/invoker `hostname | tr -dc '0-9'` >> /dev/stderr
```
In this way we can actually use the official `openwhisk/invoker` docker image.

Can this Docker `CMD` option be edited in Kube deployment as well, for those that want to reuse the official image ?